app/src/main/java/ch/epfl/sweng/rps/FriendsInfo.kt

```diff
@@ -0,0 +1,6 @@
+package ch.epfl.sweng.rps
+
+data class FriendsInfo(var username : String, var gamesPlayed : String, var gamesWon : String, var winRate : String, var onlineImage : Int)
```

Comment:
Why wwould the amount of games played be a String ? Idem for  `gamesWon`, `winRate`.